Source/rclUE/Private/ROS2Node.cpp

```diff
@@ -76,8 +77,7 @@ void AROS2Node::Init()
             Support = GetGameInstance()->GetSubsystem<UROS2Subsystem>()->GetSupport();
 
             UE_LOG(LogROS2Node, Log, TEXT("rclc_node_init_default"));
-            RCSOFTCHECK(rclc_node_init_default(
-                &node, TCHAR_TO_UTF8(*Name), Namespace != FString() ? TCHAR_TO_UTF8(*Namespace) : "", &Support->Get()));
```

Comment:
@tetorea @yuokamoto @chalice-graeme I simplified this part a bit with a bit of caution like wondering about its earlier handling, so please give any idea if you might have.
Would there be any legit difference between `""` & `TCHAR_TO_UTF8(FString())` as `const char*` param to `rclc_node_init_default()` here?  Thanks.

Reply:
@tetorea @yuokamoto @chalice-graeme I simplified this part with a bit of caution like wondering about its earlier handling, so please give any idea if you might have.
Would there be any legit difference between `""` & `TCHAR_TO_UTF8(FString())` as `const char*` param to `rclc_node_init_default()` here?  Thanks.